### ISSUE TITLE
Convert production database backup to modern version of PostgreSQL

### ISSUE THREAD

**Issue by rija** [gigascience/gigadb-website#731]
* Manually convert a production database dump into backup format supported by PostgreSQL 9.6+
* Wrap the manual process into a script so we can repeat it
* Update the deployment docs with this process.

**Comment by kencho51** [comment]
Hi @rija 

I branched off from your `fuw-cicd` branch, here is my exploration of this task:
```
% docker-compose run --rm test bash
Creating deployment_test_run ... done
root@d7be7fdbf81c:/var/www# psql -V
psql (PostgreSQL) 9.6.22
root@d7be7fdbf81c:/var/www# pg_dump --version
pg_dump (PostgreSQL) 9.6.22
root@d7be7fdbf81c:/var/www# pg_restore --version
pg_restore (PostgreSQL) 9.6.22
```
The `psql` is version `9.6+` already. 

I then `pg_dump` the `production_like` database to .sql and then check the version.
```
root@d7be7fdbf81c:/var/www/sql# pg_dump -h database -U gigadb production_like > test_production_like.sql
```
And the content of `test_production_like.sql` is :
```
--
-- PostgreSQL database dump
--

-- Dumped from database version 9.6.22
-- Dumped by pg_dump version 9.6.22

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET xmloption = content;
SET client_min_messages = warning;
SET row_security = off;

```
So, the next step would be to include above process to a script?

**Comment by only1chunts** [comment]
looks like this has been done and merged in #731 so closing this ticket
